### DISCUSSION TITLE
Tabs Component: Wrap

### DISCUSSION THREAD

**Discussion by tjhunkin** [saadeghi/daisyui#2610] [Q&A]
I see that on mobile the tabs component can be dragged using your finger, but can it wrap instead as well?

**Comment by saadeghi** [comment]
What do you mean wrap? Can you explain more?

**Reply by tjhunkin** [reply]
I see currently you can drag the tabs left and right on mobile, but users get confused and don't know that they can drag it. I just added a note to mitigate it for now. What I mean by wrap is that after say the third tab that the next one displays under it rather than off screen.
